nixos/modules/cctl.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, pkgs, config, ... }:
+let
+  inherit (lib)
+    types
+    mkOption
+    mkIf
+    mkMerge
+    mkEnableOption
+    mdDoc
+    ;
+  cfg = config.services.cctl;
+in
+{
+  options.services.cctl = {
+
+    enable = mkEnableOption (mdDoc "cctl");
+
+    package = mkOption {
+      type = types.package;
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 11101;
+      example = 60000;
+      description = mdDoc ''
+        Port to listen on.
+        TODO make port configurable in cctl
+      '';
+    };
+
+    logLevel = mkOption {
+      type = types.enum [
+        "error"
+        "warn"
+        "info"
+        "debug"
+        "trace"
+      ];
```

Comment:
Can we get a string option for "RUST_LOG".
https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives

Reply:
done [a0d71e8](https://github.com/cspr-rad/kairos/pull/46/commits/a0d71e8caa93ebe04a9c156e8b1ec749478d3b26)